requirements/test.txt

```diff
@@ -26,48 +22,50 @@ factory-boy==2.12.0
 faker==2.0.4              # via factory-boy
 filelock==3.0.12          # via tox
 freezegun==0.3.12
-funcsigs==1.0.2           # via mock, pytest
-futures==3.3.0 ; python_version == "2.7"  # via isort
```

Comment:
It looks like you ran `make upgrade` on Python 3, so it dropped all the dependencies needed for 2.7.  We should keep running that on the lowest version of Python we support to get as comprehensive of a dependency list as possible.